publication-commons/src/main/java/no/unit/nva/publication/model/business/importcandidate/NotImported.java

```diff
@@ -0,0 +1,24 @@
+package no.unit.nva.publication.model.business.importcandidate;
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import nva.commons.core.JacocoGenerated;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+public class NotImported implements ImportStatus {
```

Comment:
I honestly don't understand why, a class and an interface is used to convey a status (importstatus)

This could be done with a simple flag.

NotImported donesnt hold any information (what is not imported?)